packages/sources/coinmarketcap-test/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@chainlink/coinmarketcap-test-adapter",
+  "version": "0.0.0",
+  "description": "Chainlink coinmarketcap-test adapter.",
+  "keywords": [
+    "Chainlink",
+    "LINK",
+    "blockchain",
+    "oracle",
+    "coinmarketcap-test"
+  ],
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "repository": {
+    "url": "https://github.com/smartcontractkit/external-adapters-js",
+    "type": "git"
+  },
+  "license": "MIT",
+  "scripts": {
+    "clean": "rm -rf dist && rm -f tsconfig.tsbuildinfo",
+    "prepack": "yarn build",
+    "build": "tsc -b",
+    "server": "node -e 'require(\"./index.js\").server()'",
+    "server:dist": "node -e 'require(\"./dist/index.js\").server()'",
+    "start": "yarn server:dist"
+  },
+  "dependencies": {
+    "@chainlink/external-adapter-framework": "0.19.0",
```

Comment:
I know it's frequently changing these days... but just a reminder to use the latest version of the framework :sweat_smile:

Reply:
Looks like we had another version since. Could you bump it once more? Hopefully that's the last fw deployment before we get this merged.